subsystems/provisioning/definitions.go

```diff
@@ -29,6 +29,10 @@ const (
 	ConnCheckFilepath = "/etc/NetworkManager/conf.d/80-viam.conf"
 	ConnCheckContents = "[connectivity]\nuri=http://packages.viam.com/check_network_status.txt\ninterval=300\n"
 
+	wifiPowerSaveFilepath        = "/etc/NetworkManager/conf.d/81-viam-wifi-powersave.conf"
+	wifiPowerSaveContentsDefault = "# This file intentionally left blank.\n"
```

Comment:
If not set, we basically want to not set anything. Explicitly setting 0 or 1 might still override another file in conf.d that a user put in place.